css/style.css

```diff
@@ -1,19 +1,262 @@
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
+
+ /****** General styles *******/
+
+ body {
+
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    width: 100%;
+}
+header{
+    width: 90%;
+    margin-left: 20px;
+}
+
+/******   Navigation styles  *******/
+
+nav{
+    float: right;
+}
+a{
+    text-decoration: none;
+    color: #575556;
+    display: block;
+   
+}
+a:hover{
+    color:rgb(194, 79, 34);
+}
+.icon{
+    width: 40px;
+    height: 40px;
+    margin-left: 20px;
+    margin-top: 15px;
+}
+
+ ul{
+    display: flex;
+    flex-direction: row;
+    padding-top: 5px;
+}
+
+li{
+    padding: 10px;
+    list-style-type: none;
+}
+
+.active {
+    color: #1d1f1d;
+  }
+  .dropdown {
+    position: relative;
+    display: none;
+  }
+@media screen and (max-width: 600px) {
+    nav{
+        display: none;
+    }
+    .drop-img {
+       margin: 15px 15px 1px 10px;
+        color: white;
+        padding: 12px 15px 16px 15px;
+        font-size: 16px;
+        border: none;
+        cursor: pointer;
+      }
+      
+      .dropdown {
+          float: right;
+        position: relative;
+        display: inline-block;
+      }
+      
+      .dropdown-content {
+        display: none;
+        position: absolute;
+        background-color: #f9f9f9;
+        min-width: 120px;
+        box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+        z-index: 1;
+      }
+      
+      .dropdown-content a {
+        color: black;
+        padding: 12px 16px;
+        text-decoration: none;
+        display: block;
+      }
+      
+      .dropdown-content a:hover {
+          background-color: #f1f1f1
+        }
+      
+      .dropdown:hover .dropdown-content {
+        display: block;
+      }
+      
+      .dropdown:hover .drop-img {
+        background-color: #caceca;
+      }
+      .icon{
+          margin-left: 1px;
+          margin-bottom: 10px;
+      }
+  }
+
+/********  Hero styles    ******/
+
+.sec-background{
+    background: url('../img/first-background.jpg')  no-repeat center ;
+    background-size: cover;
+    background-clip: content-box;
+    width: 100%;
+    height: 735px;
+    color: white;
+    text-align: center;
+}
+.dv-background{
+   text-align: center;
+   transform: translate(-50%, -50%);
+
+}
```

Comment:
Try not to include the element name in your CSS class -- for example use `.intro` instead of `.p-intro`. 

If you want to restrict `.intro` to only `p` elements, you can instead use `p .intro`:

```css
p .intro {
    font-size: 3.5rem;
    /* ... */
}
```